addons/binding/org.openhab.binding.cbus/README.md

```diff
@@ -0,0 +1,17 @@
+#C-Bus Binding
```

Comment:
Could you also add sections about available channels and a textual configuration example? See [here](http://docs.openhab.org/addons/bindings/fsinternetradio/readme.html#channels) for inspiration!